scripts/changesets/changelog-generate.js

```diff
@@ -10,12 +10,14 @@ const cwd = process.cwd();
 
 // Umbrella package
 const componentsPkg = '@contentful/f36-components';
+
 // Packages we don't want to have on changelog
 const ignorePkgs = [
   '@contentful/f36-cdn',
   '@contentful/f36-i18n-utils',
   '@contentful/f36-website',
   '@contentful/f36-docs-utils',
+  '@contentful/f36-ai-components',
```

Comment:
Is this correct? If so, I have added a note to the prerelease section in `RELEASES.md` to document it.